FindModplug.cmake

```diff
@@ -9,6 +9,7 @@ find_package(PkgConfig)
 if(PKG_CONFIG_FOUND)
   pkg_check_modules (MODPLUG libmodplug)
   list(APPEND MODPLUG_INCLUDE_DIRS ${MODPLUG_INCLUDEDIR})
+  link_directories(${MODPLUG_LIBRARY_DIRS})
```

Comment:
If you want to fix this CMake find module, then do it the right way by never trusting pkg-config only.